crates/language/src/ruby.rs

```diff
@@ -59,7 +59,10 @@ impl Language for Ruby {
     }
 
     fn snippet_context_strings(&self) -> &[(&'static str, &'static str)] {
-        &[("", "")]
+        &[
+            ("", ""),
+            ("case GRIT_VARIABLE\n", "\nend")
```

Comment:
I suspect we need more contexts. Can you verify these cases work?
- Matching on a method by itself (`def $_: $_` inside a class)
- Matching an hash property (`name: "Alice"` inside `person = {name: "Alice", age: 25, city: "New York"}`)

Reply:
I updated the contexts and tests in https://github.com/getgrit/gritql/pull/286/commits/e3e4bd1436438b0cfaea7a8fc4a21d4dd2f61253#

- matching a method by itself inside a class is added in test `ruby_class_method`
- matching a hash property is added in test `ruby_hash`